### ISSUE TITLE
Support default Google client ID addition to Leo

### ISSUE THREAD

**Issue by akarukappadath** [DataBiosphere/saturn-ui#540]
Followup after this Leo ticket: https://github.com/DataBiosphere/leonardo/issues/289

You can now specify a default Google Client ID at cluster creation time in Leo to avoid the issue described above. From conversation with Isaac, it looks like the Saturn update might just be a one-line fix in ajax.js.

**Comment by zarsky-broad** [comment]
@akarukappadath What's the exact name for the label? I can't seem to find it.

**Comment by zarsky-broad** [comment]
If this causes the authentication to happen automatically, we should remove the `setToken` call every 20 minutes as part of this.
